dallinger/frontend/templates/dashboard_database.html

```diff
@@ -192,6 +192,17 @@ <h1>{{ title }}</h1>
           copyBox(commands.join('; '));
         }
       };
+
+      function updateBooleanProperties(data) {
+        const booleanProperties = ["aborted", "failed", "complete", "main_consent"]
```

Comment:
Can you explain what this recoding achieves in terms of the display? I don't think we should hard-code which columns need recoding. Maybe it's better to infer whether the column is boolean by reading the values in it?

Reply:
Filtering on boolean fields using the search panes didn't work unless their boolean values were converted to strings. Some peolple reported similar issues, e.g. here: https://datatables.net/forums/discussion/73232/best-way-to-implement-full-boolean-data-type-support-searchbuilder-server-side-processing

Reply:
Properties are now auto-detected whether they are of type boolean.

Reply:
The new code looks good to me. I think you should add a comment to the code though explaining what you just described to me.

Reply:
:100:
Done